src/Serilog.Sinks.Seq/Sinks/Seq/Audit/SeqAuditSink.cs

```diff
@@ -52,8 +51,8 @@ async Task EmitAsync(LogEvent logEvent)
         if (logEvent == null) throw new ArgumentNullException(nameof(logEvent));
 
         var payload = new StringWriter();
-        CompactJsonFormatter.FormatEvent(logEvent, payload, JsonValueFormatter);
+        _payloadFormatter.Format(logEvent, payload);
```

Comment:
There's a slight behavior change here; audit payloads will carry a redundant trailing `\n` that wasn't emitted previously. I don't think it's worth going to any effort to remove this right now.